dataflow/src/main/java/org/checkerframework/dataflow/cfg/ControlFlowGraph.java

```diff
@@ -124,14 +129,38 @@ public SpecialBlock getEntryBlock() {
         return entryBlock;
     }
 
+    /**
+     * Returns all return nodes of the control flow graph.
+     *
+     * @return all return nodes of the control flow graph.
+     */
     public List<ReturnNode> getReturnNodes() {
         return returnNodes;
     }
 
+    /**
+     * Return the top-level nodes of all the expression statements encountered.
```

Comment:
To be consistent, here maybe also use `Returns`?

Reply:
Okay, thanks for pointing out!